### PR TITLE
Feature/dark mode

### PR DESCRIPTION
**Pull request by Mortadha-abderrhim**
Make the app ready to support the dark theme.

Screenshot_1617879525
Screenshot_1617877720
Screenshot_1617877732
Screenshot_1617879548

Disable the dark theme for now.

### CODE REVIEW

File: app/src/main/java/com/pindex/main/ui/fragments/blocks/BigHeaderBlockFragment.kt [hunk 1]
```diff
@@ -22,6 +23,9 @@ class BigHeaderBlockFragment : Fragment(R.layout.fragment_pindex_block_big_heade
         // Set the text
         textView.text =
             arguments?.getString(Constants.BlockFragmentArguments.BIG_HEADER_BLOCK_TITLE)
+        textView.setTextColor(AppCompatResources.getColorStateList(requireContext(),R.color.material_on_background_emphasis_high_type))
```
Comment (lucasrlt) [inline app/src/main/java/com/pindex/main/ui/fragments/blocks/BigHeaderBlockFragment.kt:26]: This line looks weird, maybe needs formatting ?

File: app/src/main/java/com/pindex/main/ui/fragments/blocks/SectionTitleBlockFragment.kt [hunk 2]
```diff
@@ -22,6 +24,7 @@ class SectionTitleBlockFragment : Fragment(R.layout.fragment_pindex_block_sectio
         // Set the text
         textView.text =
             arguments?.getString(Constants.BlockFragmentArguments.SECTION_TITLE_BLOCK_TITLE)
+        textView.setTextColor(AppCompatResources.getColorStateList(requireContext(),R.color.material_on_background_emphasis_high_type))
```
Comment (lucasrlt) [inline app/src/main/java/com/pindex/main/ui/fragments/blocks/SectionTitleBlockFragment.kt:27]: Same here

**Review by lucasrlt — COMMENTED** [review]
Great use of the attributes for the color theme !
The two files I commented could use a small code reformatting (or is it just github?), but otherwise everything is perfect

File: app/src/main/java/com/pindex/main/ui/fragments/blocks/BigHeaderBlockFragment.kt (hunk 1 above)
Comment (Mortadha-abderrhim) [reply app/src/main/java/com/pindex/main/ui/fragments/blocks/BigHeaderBlockFragment.kt]: Changing the color in the XML file caused an inflating problem in the build. So, I had to do it programmatically after the text view is inflated. I reformatted the code for further readability.

**Review by lucasrlt — APPROVED** [review]
Reformatting looks great, nice job !

**Review by aouame — APPROVED** [review]
After reformatting code, it looks good !

**Pull request merged**